pool/tick_bitmap_test.gno

```diff
@@ -0,0 +1,239 @@
+package pool
+
+import (
+	"testing"
+
+	u256 "gno.land/p/gnoswap/uint256"
+)
+
+func TestTickBitmapPosition(t *testing.T) {
+	tests := []struct {
+		name        string
+		tick        int32
+		wantWordPos int16
+		wantBitPos  uint8
+	}{
+		{
+			name:        "positive tick",
+			tick:        300,
+			wantWordPos: 1,  // 300 >> 8 = 1
+			wantBitPos:  44, // 300 % 256 = 44
+		},
+		{
+			name:        "negative tick",
+			tick:        -300,
+			wantWordPos: -2,  // -300 >> 8 = -2
+			wantBitPos:  212, // -300 % 256 = 212
+		},
+		{
+			name:        "zero tick",
+			tick:        0,
+			wantWordPos: 0,
+			wantBitPos:  0,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			wordPos, bitPos := tickBitmapPosition(tt.tick)
+			if wordPos != tt.wantWordPos {
+				t.Errorf("wordPos = %v, want %v", wordPos, tt.wantWordPos)
+			}
+			if bitPos != tt.wantBitPos {
+				t.Errorf("bitPos = %v, want %v", bitPos, tt.wantBitPos)
+			}
+		})
+	}
+}
+
+func TestTickBitmapFlipTick(t *testing.T) {
+	tests := []struct {
+		name        string
+		tick        int32
+		tickSpacing int32
+		shouldPanic bool
+	}{
+		{
```

Comment:
Need to add negative value test case

Reply:
c4b5a28711e09d74d36008fae5faf45a3acaa928